lib/timeval.c

```diff
@@ -40,6 +40,7 @@ struct curltime Curl_now(void)
                         Curl_freq.QuadPart);
   }
   else {
+#ifndef CURL_WINDOWS_APP
```

Comment:
should probably #ifndef CURL_WINDOWS_APP on the vista check. Static analysis/compiler warnings might fire off and say that you are potentially returning an uninitialized variable.